package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@octokit/plugin-throttling.js",
+  "version": "0.0.1",
+  "description": "Automatic rate limiting plugin for octokit",
+  "main": "lib/index.js",
+  "scripts": {
+    "pretest": "standard",
+    "test": "npx mocha test/"
```

Comment:
why npx?

Reply:
That was an oversight and it's already fixed in a later commit. I think you may be looking at an outdated page.